source/pymongo.txt

```diff
@@ -76,10 +75,24 @@ Connect to MongoDB Atlas
 
    import pymongo
 
-   client = pymongo.MongoClient(
-      "mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority")
-   db = client.test
+   // replace this with your MongoDB connection string
+   conn_str = "<your MongoDB Atlas connection string>"
+
+   client = pymongo.MongoClient(conn_str, serverSelectionTimeoutMS=5000)
+
+   try:
+       print(client.server_info())
+   except Exception:
+       print("Unable to connect to the server.")
+
+If the connection succeeds before a five-second timeout, you should see a
+dictionary containing information about the server you connected to.
+
+If the connection fails, you should see the message "Unable to connect to
```

Comment:
Suggestion: Since this is output, can it be in a code-block?

Also applies to Motor page.

Reply:
I'll try it out. I intentionally did not format it in a code-block because:
1) a code block is more visually prominent than the description I'm using for the cse of successful output in the prior paragraph
2) it takes up more vertical space, and much more vertical space if I tried to describe the output iof server_info.